azurelinuxagent/ga/update.py

```diff
@@ -324,20 +324,11 @@ def run(self, debug=False):
         """
 
         try:
+            logger.info("{0} Version: {1}", AGENT_LONG_NAME, CURRENT_AGENT)
+            logger.info("OS: {0} {1}", DISTRO_NAME, DISTRO_VERSION)
```

Comment:
NIT: Repeated logs (Line 328-329 are already being logged in line 345)

Reply:
Yes, this repeats some of the info on L345, but removing the duplicated info is not worth it. Also, the intention is to make it as close as possible to the same message logged by the Daemon. And L345 is also send to telemetry.